.rubocop

```diff
@@ -0,0 +1 @@
+--ignore-parent-exclusion
```

Comment:
This forces the root project to not exclude any files so RuboCop can be correctly run on the `shared` directory